README.md

```diff
@@ -33,11 +57,13 @@ and uses it as the sample name. Further control is possible with:
 #### "version_2"
 |Switch                                   | Default | Description |
 |---|---|---|
-|matchExactSampleName                     |false    | If set, the sample will be extract like they are set in the config. This is compatible with allowSampleTerminationWithIndex. |
+|matchExactSampleName                     |false    | If set, the sample will be extracted like they are set in the config. This is compatible with allowSampleTerminationWithIndex. |
 |allowSampleTerminationWithIndex          |true     | Allow recognition of trailing integer numbers for sample names, e.g. tumor_02, if tumor is set. |
 |useLowerCaseFilenamesForSampleExtraction |true     | The switch will tell the method to work on lowercase filenames.|
 
-Describe matching strategy
+Please take a close look at the file SampleFromFilenameExtractorVersionTwoTest (search with find or so). There is 
```

Comment:
consider making a relative link into the code here

Reply:
not relative, but its done.